client/src/pages/admin/MergePeople.js

```diff
@@ -1,388 +1,883 @@
+                dispatchMergeActions={dispatchMergeActions}
+              />
+              <PersonField
+                label="Biography"
+                value={parseHtmlWithLinkTo(mergedPerson.biography)}
+                align={ALIGN_OPTIONS.CENTER}
+                action={getClearButton(() =>
+                  dispatchMergeActions(setAMergedField("biography", "", null))
+                )}
+                fieldName="biography"
+                mergeState={mergeState}
+                dispatchMergeActions={dispatchMergeActions}
+              />
+              {Settings.fields.person.customFields &&
+                Object.entries(Settings.fields.person.customFields).map(
+                  ([fieldName, fieldConfig]) => {
+                    const fieldValue =
+                      mergedPerson?.[DEFAULT_CUSTOM_FIELDS_PARENT]?.[fieldName]
+                    return (
+                      <PersonField
+                        key={fieldName}
+                        label={fieldConfig.label || fieldName}
+                        value={JSON.stringify(fieldValue)}
+                        align={ALIGN_OPTIONS.CENTER}
+                        action={getClearButton(() =>
+                          dispatchMergeActions(
+                            setAMergedField(
+                              `${DEFAULT_CUSTOM_FIELDS_PARENT}.${fieldName}`,
+                              CUSTOM_FIELD_TYPE_DEFAULTS[fieldConfig.type],
+                              null
+                            )
+                          )
+                        )}
+                        fieldName={`${DEFAULT_CUSTOM_FIELDS_PARENT}.${fieldName}`}
+                        mergeState={mergeState}
+                        dispatchMergeActions={dispatchMergeActions}
                       />
-                    )}
-                    {loser &&
-                      !_isEmpty(loser.position) &&
-                      winner &&
-                      !_isEmpty(winner.position) && (
-                        <Alert variant="danger">
-                          <b>Danger:</b> Position on Loser (
-                          {loser.position.name}) will be left unfilled
-                        </Alert>
-                    )}
-                  </Col>
-                </Row>
-                <Row>
-                  <Col md={12}>
-                    <Button
-                      style={{ width: "98%", margin: "16px 1%" }}
-                      variant="primary"
-                      onClick={submitForm}
-                      disabled={isSubmitting}
-                    >
-                      Merge People
-                    </Button>
-                  </Col>
-                </Row>
-              </Container>
-            </Form>
-          )
-        }}
-      </Formik>
-    </div>
+                    )
+                  }
+                )}
+            </>
+          )}
+        </Col>
+        <Col md={4} id="right-merge-per-col">
```

Comment:
Please put a fieldset in the background to be consistent with the rest of the application.